.github/workflows/deploy-sbx.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy-SBX
+
+on:
+  push:
+    branches: [ feat/sbox-rds ]
+
+permissions:
+  contents: read
+
+jobs:
+  EC2-Deploy:
+    # if the branch name of the PR does not contain 'skip-deploy'
+    if: "!contains(github.head_ref, 'skip-deploy')"
+    runs-on: ubuntu-latest
+    environment:
+      name: ${{ github.ref_name }}
+      url: ${{ steps.deploy.outputs.vm_url }}
+    steps:
+    - id: deploy
+      name: Deploy
+      uses: bitovi/github-actions-deploy-docker-to-ec2@v0.0.4.9
+      with:
+        aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID_SANDBOX}}
+        aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY_SANDBOX}}
+        aws_default_region: us-east-1
+        tf_state_bucket: bitovi-operations-aws-sandbox
+        aws_resource_identifier: bitovi-gha-deploy-docker-to-ec2
+
+        # Provide a secret called `DOT_ENV` to append environment variables to the .env file
+        # dot_env: ${{ secrets.DOT_ENV }}
+
```

Comment:
we're not configuring RDS anywhere in here.  there should at least be something like:
`enable_rds: true`